### DISCUSSION TITLE
Why is not the response data up-to-date immediately whenever I re-invoke the method?

### DISCUSSION THREAD

**Discussion by technqvi** [fastapi/fastapi#6381] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
import pandas as pd

app = FastAPI()

@app.get("/")
async def root():
    return {"message": "Hello Pycharm"}

@app.get("/sitegrade")
async def list_site_grade():
    site_grade_data = {
        'company': ['AIS', 'BBL', 'BDMS', 'AIS', 'BBL', 'BDMS'],
        'quarter': [1, 1, 1, 2, 2, 2],
        'year': [2022, 2022, 2022, 2022, 2022, 2022],
        'no_server_unit': [5, 3, 7, 4, 4, 2],
        'size_storage': [500, 300, 700, 400, 400, 200],
        'no_request': [2, 5, 7, 4, 5, 2],
        'no_incident': [20, 35, 70, 40, 25, 20],
        'score_server': [10, 15, 20, 10, 5, 5],
        'score_storage': [15, 20, 30, 10, 8, 7],
        'score_request': [20, 30, 30, 10, 10, 45],
        'score_incident': [15, 25, 20, 5, 5, 20]

    }
    df = pd.DataFrame(data=site_grade_data)
    # df['total_score'] = df['score_server'] + df['score_storage'] + df['score_request'] + df['score_incident']
    result = df.to_dict('records')
    return result
```

### Description

Hi 
The above code is sample data, I apply the pandas' data frame to generate sample data and return it as json file.

After I try invoking the method at the first time and I edit the data and invoke again. 
I doubt  why the response data on browser is not updated according to data updated on the server.

I  run the server as this command and try to invoke the method as the figure below.
 uvicorn main:app --host 127.0.0.1 --port 9999
http://127.0.0.1:9999/sitegrade

this is reponse data at the first invoking.
<img width="201" alt="image" src="https://user-images.githubusercontent.com/38780060/165936207-0712bb55-2f54-454a-9199-fd0484121b5b.png">

after that, I try to change the data in the dictionary variable in code as figure below.
![image](https://user-images.githubusercontent.com/38780060/165936571-602adf71-c1b5-4387-809b-93e8b82edc20.png)

but this is nothing changes in response data. Indeed,  it should change from "AIS" to "AIS555" in the company as the figure below.
<img width="194" alt="image" src="https://user-images.githubusercontent.com/38780060/165936982-658fce3d-7de1-4ba0-a793-4310cea4a601.png">

Please guide me how to fix this problem.
I am new for this framework.

### Operating System

Windows

### Operating System Details

Processor	Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz   2.30 GHz
Installed RAM	16.0 GB (15.8 GB usable)
System type	64-bit operating system, x64-based processor
Pen and touch	Touch support with 2 touch points

### FastAPI Version

fastapi 0.75.2

### Python Version

python 3.9

### Additional Context

**I want to apply FastAPI to do the project in the following detail.**

1. Create API service by retrieving data from PostgreSQL database
2. Use Pandas data frame to do something with the data properly and return it as API service.
3. Apply PowerBI to get data from FAST API to create charts/graphs on dashboard.
4. Schedule on a daily basis  for refreshing   updated data from Server via Fast-API

**Comment by accelleon** [comment, marked as answer]
If you want to make live changes to the code without manually restarting the server you need to specify `--reload` as uvicorn parameter. All it does it watch your source files for modifications and reload the server with the new code. Do *not* do this for any form of deployment, this is purely for a faster dev environment.
```
uvicorn main:app --host 127.0.0.1 --port 9999 --reload
```

**Comment by technqvi** [comment]
> 

Thank you for your prompt reply.
